cmd/signozcollectormigrator/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# use a minimal alpine image
+FROM alpine:3.17
+
+# add ca-certificates in case you need them
+RUN apk update && apk add ca-certificates && rm -rf /var/cache/apk/*
+
+# define arguments and default values
+ARG TARGETOS TARGETARCH
+ARG USER_UID=10001
+
+# create a non-root user for running the migrator
+USER ${USER_UID}
+
+# copy the binaries from the multi-stage build
+COPY .build/${TARGETOS}-${TARGETARCH}/signoz-collector-migrator /signoz-collector-migrator
+
+# run the binary as the entrypoint and pass the default dsn as a flag
+ENTRYPOINT [ "/signoz-collector-migrator" ]
+CMD ["--dsn", "tcp://host.docker.internal:9000"]
```

Comment:
@srikanthccv this change was intentional..since we are expecting people to run images built by this dockerfile on containers, localhost:9000 might not point to a clickhouse server..in such a case host.docker.internal:9000 is a better default value as it uses the docker host's port 9000 which is much more likely to be bound to a clickhouse server(either running natively or port forwarded from a container)

Reply:
This work is more important in other environments than the default case it tries to handle. You may have had this convenient for testing, but in the real world, this will be used in k8s environments 90% of the time. For this reason, I believe this docker networking specific _running in a container trying to access host ClickHouse_ shouldn't be the default.